src/main/java/TaskList.java

```diff
@@ -0,0 +1,232 @@
+import duke.*;
+
+public class TaskList {
+    //private Storage storage;
+
+    public TaskList() {
+        //storage = new Storage();
```

Comment:
Avoid leaving out commented code in the program, it reduces readability. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#delete-dead-code)